### PR TITLE
Added instances for competency question #51.

### PR DESCRIPTION
**Pull request by sofimrtn**
Added two new researchers and two publications for each one of them, as well as a new edited publication. Also added roles to existing researchers.

### CODE REVIEW

**Comment by thewillyhuman** [comment]
@alejgh can you please review?

**Comment by thewillyhuman** [comment]
BTW is there any issue relates to this PR?

File: test/asio-individuals.ttl [hunk 1]
```diff
@@ -454,3 +459,70 @@ asioIndividuals:WESO rdf:type owl:NamedIndividual ,
...
+                                    "Maria Rita Sierra Sanchez"@es .
+
+###  http://www.asio.es/asioinstances/asio#MariaRitaPaper1
+asioIndividuals:MariaRitaPaper1 rdf:type owl:NamedIndividual ,
+                               asio:PeerReviewedArticle ;
+                      asio:isAuthoredBy asioIndividuals:MariaRitaSierraSanchez ;
+                      asio:publicationDate 2015 ;
+                      asio:publishedBy asioIndividuals:Springer;
+                      rdfs:comment "Mencía R., Sierra M.R., Varela R. (2015) Genetic Algorithm for the Job-Shop Scheduling with Skilled Operators. 
+                                    In: Ferrández Vicente J., Álvarez-Sánchez J., de la Paz López F., Toledo-Moreo F., Adeli H. (eds) Bioinspired Computation in
```
Comment (alejgh) [inline test/asio-individuals.ttl:488]: This is not valid in Turtle. In order to write multiline literals you need to use triple quotes 
 (either ''' or """). For example:
```turtle
:Example rdfs:comment """This is
                  a multiline
                  string."""@en
```

If you use single quotes you can't write a multiline string  so all the literals with multiple lines (like the one above) can either be written in a single line in the editor or stay the same and replace the single quotes by triple quotes.

More information can be found here if needed: https://www.w3.org/TR/turtle/#turtle-literals

File: test/asio-individuals.ttl [hunk 2]
```diff
@@ -454,3 +459,70 @@ asioIndividuals:WESO rdf:type owl:NamedIndividual ,
...
+###  http://www.asio.es/asioinstances/asio#MariaRitaPaper1
+asioIndividuals:MariaRitaPaper1 rdf:type owl:NamedIndividual ,
+                               asio:PeerReviewedArticle ;
+                      asio:isAuthoredBy asioIndividuals:MariaRitaSierraSanchez ;
+                      asio:publicationDate 2015 ;
+                      asio:publishedBy asioIndividuals:Springer;
+                      rdfs:comment "Mencía R., Sierra M.R., Varela R. (2015) Genetic Algorithm for the Job-Shop Scheduling with Skilled Operators. 
+                                    In: Ferrández Vicente J., Álvarez-Sánchez J., de la Paz López F., Toledo-Moreo F., Adeli H. (eds) Bioinspired Computation in 
+                                    Artificial Systems. IWINAC 2015. Lecture Notes in Computer Science, vol 9108. Springer, Cham"@en;
+                      rdfs:label "Genetic Algorithm for the Job-Shop Scheduling with Skilled Operators"@en;
```
Comment (alejgh) [inline test/asio-individuals.ttl]: This should end with a '.' and not a ';'

File: test/asio-individuals.ttl [hunk 3]
```diff
@@ -454,3 +459,70 @@ asioIndividuals:WESO rdf:type owl:NamedIndividual ,
...
+###  http://www.asio.es/asioinstances/asio#BegoñaCristinaPelayoPaper1
+asioIndividuals:BegoñaCristinaPelayoPaper1 rdf:type owl:NamedIndividual ,
+                               asio:PeerReviewedArticle ;
+                      asio:isAuthoredBy asioIndividuals:BegoñaCristinaPelayoGarciaBustelo ;
+                      asio:publishedBy asioIndividuals:Springer;
+                      asio:publicationDate 2010 ;
+                      rdfs:comment "Álvarez M.Á., G-Bustelo B.C.P., Sanjuán-Martínez O., Lovelle J.M.C. (2010) Bridging together Semantic Web and Model-Driven 
+                                    Engineering. In: de Leon F. de Carvalho A.P., Rodríguez-González S., De Paz Santana J.F., Rodríguez J.M.C. (eds) Distributed 
+                                    Computing and Artificial Intelligence. Advances in Intelligent and Soft Computing, vol 79. Springer, Berlin, Heidelberg"@en;
+                      rdfs:label "Bridging together Semantic Web and Model-Driven Engineering"@en;
```
Comment (alejgh) [inline test/asio-individuals.ttl]: End with '.'

File: test/asio-individuals.ttl [hunk 4]
```diff
@@ -454,3 +459,70 @@ asioIndividuals:WESO rdf:type owl:NamedIndividual ,
...
+                                    In: Ferrández Vicente J., Álvarez-Sánchez J., de la Paz López F., Toledo-Moreo F., Adeli H. (eds) Bioinspired Computation in 
+                                    Artificial Systems. IWINAC 2015. Lecture Notes in Computer Science, vol 9108. Springer, Cham"@en;
+                      rdfs:label "Genetic Algorithm for the Job-Shop Scheduling with Skilled Operators"@en;
+
+###  http://www.asio.es/asioinstances/asio#MariaRitaPaper2
+asioIndividuals:MariaRitaPaper2 rdf:type owl:NamedIndividual ,
+                               asio:PeerReviewedArticle ;
+                      asio:isAuthoredBy asioIndividuals:MariaRitaSierraSanchez ;
+                      asio:publicationDate 2015 ;
+                      rdfs:comment "Mencía, Carlos et al. ‘Solving the Job Shop Scheduling Problem with Operators by Depth-first Heuristic
```
Comment (alejgh) [inline test/asio-individuals.ttl]: Multiline.

File: test/asio-individuals.ttl [hunk 5]
```diff
@@ -454,3 +459,70 @@ asioIndividuals:WESO rdf:type owl:NamedIndividual ,
...
+                      rdfs:label "Solving the Job Shop Scheduling Problem with Operators by Depth-first Heuristic 
+                                    Search Enhanced with Global Pruning Rules"@en .
+
+###  http://www.asio.es/asioinstances/asio#BegoñaCristinaPelayoPaper1
+asioIndividuals:BegoñaCristinaPelayoPaper1 rdf:type owl:NamedIndividual ,
+                               asio:PeerReviewedArticle ;
+                      asio:isAuthoredBy asioIndividuals:BegoñaCristinaPelayoGarciaBustelo ;
+                      asio:publishedBy asioIndividuals:Springer;
+                      asio:publicationDate 2010 ;
+                      rdfs:comment "Álvarez M.Á., G-Bustelo B.C.P., Sanjuán-Martínez O., Lovelle J.M.C. (2010) Bridging together Semantic Web and Model-Driven
```
Comment (alejgh) [inline test/asio-individuals.ttl]: Multiline

File: test/asio-individuals.ttl [hunk 6]
```diff
@@ -454,3 +459,70 @@ asioIndividuals:WESO rdf:type owl:NamedIndividual ,
...
+                                    Computing and Artificial Intelligence. Advances in Intelligent and Soft Computing, vol 79. Springer, Berlin, Heidelberg"@en;
+                      rdfs:label "Bridging together Semantic Web and Model-Driven Engineering"@en;
+
+###  http://www.asio.es/asioinstances/asio#BegoñaCristinaPelayoPaper2
+asioIndividuals:BegoñaCristinaPelayoPaper2 rdf:type owl:NamedIndividual ,
+                               asio:PeerReviewedArticle ;
+                      asio:isAuthoredBy asioIndividuals:BegoñaCristinaPelayoGarciaBustelo ;
+                      asio:publishedBy asioIndividuals:Springer;
+                      asio:publicationDate 2010 ;
+                      rdfs:comment "Núñez V E.R., Martínez O.S., Lovelle J.M.C., García-Bustelo B.C.P. (2010) First Steps towards Implicit Feedback for
```
Comment (alejgh) [inline test/asio-individuals.ttl]: Multiline

**Review by alejgh — CHANGES_REQUESTED** [review]
There are some minor mistakes in the file that make it syntactically incorrect. I forgot to tell you that you can use a turtle validator before submitting the changes to verify that everything is correct, so this was mainly my fault.

For example, you can use this one: http://ttl.summerofcode.be

Please make the requested changes and verify that everything validates before proceeding to insert this information in our Wikibase instance.

Regarding the content of the new instances, everything seems fine to me.

**Comment by alejgh** [comment]
> BTW is there any issue relates to this PR?

This closes CQ038 associated to the issue #39 

I think we should change the title of the PR, since it not only involves adding the instances but also creating the SPARQL query to solve the CQ (see workflow below).

The workflow is as follows:
* @sofimrtn will add real instance data to the asio-instances file.
* I will verify that everything is ok (both from a syntactic and semantic POV).
* @sofimrtn will proceed to add the new instances to our Wikibase.
* I will work on the SPARQL query to 'solve' the CQ using the data of our Wikibase.
* The SPARQL query is added to the PR.
* The PR is closed after review and the issue associated to it is closed too.

My idea is to keep the same steps for following PRs associated to competency questions, but maybe the person that works on each step changes (e.g. sometimes I will work on adding the instances to the file and Wikibase and maybe @labra or @sofimrtn work with the SPARQL query...).

@thewilly do you think that we should make separate PRs to insert the instances and to develop the SPARQL query? I usually like to make a PR per issue, and that's why my original idea was to do both things in this PR, but I'm open to suggestions.

**Comment by thewillyhuman** [comment]
No no problem with that, just asking to link to the issue but one PR can be linked to multiple issues and viceversa

**Comment by sofimrtn** [comment]
Fixed syntax errors. Turtle validator confirms. Copy-pasting does not work too good sometimes when it comes to semicolons. Also thanks for the triple quotation info, had no idea :)

**Review by alejgh — APPROVED** [review]
Ok, everything looks good to me. We can now move on to add this content to our wikibase instance (https://hercules-demo.wiki.opencura.com/wiki/Main_Page). Please don't close this PR until I add the SPARQL query

**Comment by alejgh** [comment]
I have created a SPARQL query to solve the CQ: https://tinyurl.com/y95hnf8g

If you allow me write access to your fork, I can update the PR with the query and close this.

**Comment by thewillyhuman** [comment]
@alejgh you are able to add more commits to sofimrtn/hercules-ontology.

**Comment by alejgh** [comment]
> @alejgh you are able to add more commits to sofimrtn/hercules-ontology.

Yes, I needed her to add me as a collaborator in her fork. Now everything is ready.

**Comment by alejgh** [comment]
I have moved the remaining competency questions to the doc folder. This will also close #102

**Pull request merged**